### DISCUSSION TITLE
Trouble with PM tutorial code and files.. is it me?

### DISCUSSION THREAD

**Discussion by doroft** [LAPKB/Pmetrics#132] [General]
Hi there guys ... Well done on Pmetrics 2.0.2 I have been working throught the PM_tutorial() and have found a couple of issues. 

The install was with R4.2.2 and Rstudio 2022.12.0 with Rtools 42 on Windows 11  + dealing to all the packages that needed updating.

Everything is fine till the model modification with the s6 method in excercise 2
Line 302 exFit2$check() throws a fortran error in the updated model line 270-291 ... I cant see the problem 

In the Sim Runs (excercise 4)
Line 336  Sim run 3 not sub-setable

(Excercise 5) - Model validation 
  > valid_2 <- exRes2$validate(limits = NA) 
Error in missing(data) : 'missing' can only be used for arguments

The IT2B run works well now which is great. 

The excercise on PTA throws errors (Below is quoted out of the tutorial with the data) 
> pta1_2 <- PM_pta$new(
+   simdata = simlist1,
+   targets = c(0.25, 0.5, 1, 2, 4, 8, 16, 32), target.type = "time", 
+   success = 0.6, start = 120, end = 144
+ )
Error in makePTA(simdata, targets, target.type, ...) : 
  Simulation output (simdata), targets, target.type, and success are all mandatory.

I am not sure if some of these are my setup. The install was not smooth but complete in the end. The data and the text is straight out of the PM_tutorial() running the examples in the  examples.R file. I have spent a bit of time looking at these issues.

Is there anything I am doing wrong here ..... with the demonstration examples I cant think i am the issue... However I am only a novice at Pmetrics... 

Any help appreciated and thanks again.

**Comment by Siel** [comment]
Hi @doroft,
Thanks for reporting back, Could you please reinstall Pmetrics? I just tried to reproduce your error and everything worked correctly. Maybe some files were not updated correctly when you installed Pmetrics last time.
